From: stern@rowland.harvard.edu
Subject: [PATCH] USB: Make use_both_schemes=y the default

[PATCH] USB: Make use_both_schemes=y the default

Enough people are experiencing problems with the new device initialization
scheme that it seems like a good idea to make the default behavior be to
try the old scheme when the new one fails.  That's what this patch does,
simply by changing the initial value of the use_both_schemes module
parameter.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/core/hub.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/drivers/usb/core/hub.c b/drivers/usb/core/hub.c
--- a/drivers/usb/core/hub.c	2005-03-08 16:54:56 -08:00
+++ b/drivers/usb/core/hub.c	2005-03-08 16:54:56 -08:00
@@ -74,7 +74,7 @@
 MODULE_PARM_DESC(old_scheme_first,
 		 "start with the old device initialization scheme");
 
-static int use_both_schemes = 0;
+static int use_both_schemes = 1;
 module_param(use_both_schemes, bool, S_IRUGO | S_IWUSR);
 MODULE_PARM_DESC(use_both_schemes,
 		"try the other device initialization scheme if the "
